packages/Webkul/Core/src/Helpers/Laravel5Helper.php

```diff
@@ -353,50 +279,66 @@ private function createDownloadableLink(int $productId): void
         ]);
     }
 
-    /**
-     * @param int   $productId
-     * @param array $attributeValues
-     *
-     * @return void
-     */
-    private function createAttributeValues(int $productId, array $attributeValues = []): void
+    private function createAttributeValues(Product $product, array $attributeValues = []): void
```

Comment:
What do we need the whole product object for? I only found a usage of $product->id. So I would suggest to use an integer $productID in functions head.

Reply:
fixed.